crates/beerus-core/examples/call.rs

```diff
@@ -0,0 +1,49 @@
+use beerus_core::{
+    config::Config,
+    lightclient::{
+        beerus::BeerusLightClient, ethereum::helios_lightclient::HeliosLightClient,
+        starknet::StarkNetLightClientImpl,
+    },
+};
+use env_logger::Env;
+use eyre::Result;
+use starknet::{
+    core::types::FieldElement,
+    providers::jsonrpc::models::{BlockId, FunctionCall},
+};
+use std::str::FromStr;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    env_logger::Builder::from_env(Env::default().default_filter_or("info")).init();
+
+    let config = Config::from_env();
+    let ethereum_lightclient = HeliosLightClient::new(config.clone()).await?;
+    let starknet_lightclient = StarkNetLightClientImpl::new(&config)?;
+    let mut beerus = BeerusLightClient::new(
+        config.clone(),
+        Box::new(ethereum_lightclient),
+        Box::new(starknet_lightclient),
+    );
+    beerus.start().await?;
+
+    let calldata = FunctionCall {
+        contract_address: FieldElement::from_str(
+            "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
```

Comment:
I would have used variables to make the example more explicit about the call. What do you think about:

```rust
let starkgate_addr = "0x049...c7";
let name_selector = "0x036...60";

let calldata = FunctionCall {
    contract_address: FieldElement::from_str(starkgate_addr)
    .unwrap(),
    entry_point_selector: FieldElement::from_str(name_selector)
    .unwrap(),
    calldata: vec![],
};

```